src/Microsoft.ML.AutoML/Experiment/ModelContainer.cs

```diff
@@ -30,8 +31,10 @@ internal ModelContainer(MLContext mlContext, FileInfo fileInfo, ITransformer mod
             }
         }
 
-        public ITransformer GetModel()
+        public ITransformer GetModel(bool isModelDisposed)
         {
```

Comment:
The model is owned by the ModelContainer.   Shouldn't it know whether it was disposed or not?